etl/steps/data/meadow/happiness/2023-03-20/happiness.meta.yml

```diff
@@ -0,0 +1,37 @@
+dataset:
+  title: World Happiness Report
+  description: |-
+    The World Happiness Report reflects a worldwide demand for more attention to happiness and well-being as criteria for government policy. It reviews the state of happiness in the world today and shows how the science of happiness explains personal and national variations in happiness.
+
+    Life evaluations from the Gallup World Poll provide the basis for the annual happiness rankings. They are based on answers to the main life evaluation question. The Cantril ladder asks respondents to think of a ladder, with the best possible life for them being a 10 and the worst possible life being a 0. They are then asked to rate their own current lives on that 0 to 10 scale. The rankings are from nationally representative samples over three years.
+  sources:
+  - name: World Happiness Report (2023)
+  - published_by: 'Helliwell, J. F., Layard, R., Sachs, J. D., De Neve, J.-E., Aknin, L. B., & Wang, S. (Eds.). (2023). World Happiness Report 2023. New York: Sustainable Development Solutions Network.'
+  - publisher_source: Data comes from the Gallup World Poll and the Gallup/Meta State of Social Connections study.
+  - url: https://worldhappiness.report
+  - date_accessed: 2023-03-20
+tables:
+  happiness:
+    title: World Happiness Report
+    variables:
+      country:
+        title: Country name
+        short_unit: ''
+        unit: ''
```

Comment:
I haven't checked, but if we set `country` (along with `year`) as indices in `garden`, does this disappear? Like, there will not be a "country" variable.